src/App.js

```diff
@@ -4,12 +4,14 @@ import './App.css';
 import Auth from './ui/containers/auth.container';
 import Users from './ui/containers/users.container';
 import Books from './ui/containers/books.container';
+import SearchBox from './ui/containers/book-search.container';
 
 const App = () =>
   <div className="App">
     <div className="App-header">
       <img src={logo} className="App-logo" alt="logo" />
       <h2>Book App</h2>
+      <SearchBox />
     </div>
     <div className="mainContent">
       <Auth />
```

Comment:
Uses the "connected" search box component. Not the dumb one.
Note the `.container`